Storage/tests/Unit/StreamWrapperTest.php

```diff
@@ -25,12 +25,11 @@
 use Google\Cloud\Storage\StreamWrapper;
 use GuzzleHttp\Psr7;
 use Prophecy\Argument;
-use Prophecy\Promise\CallbackPromise;
-use Prophecy\Prophecy\ObjectProphecy;
 use PHPUnit\Framework\TestCase;
 
 /**
  * @group storage
+ * @group storage-streamwrapper
```

Comment:
```suggestion
 * @group storage-stream-wrapper
```

Small nit: Just to match the other annotations